### PR TITLE
[#194] Document how to deploy Liferay applications into a Liferay service

### PR DESCRIPTION
**Pull request by mdelapenya**
This PR requires https://github.com/wedeploy/images/pull/437 and https://github.com/wedeploy/boilerplate-liferay/pull/2 to be merged and deployed

### CODE REVIEW

File: ui/src/pages/docs/other/liferay.md [hunk 1]
```diff
@@ -57,6 +57,20 @@ The `id` for your services are uniquely determined by you.
 
 </article>
 
+<article id="4">
+
+## Deploying Liferay applications
+
+Deploying Liferay applications into a Liferay service is super straightforward. Just create a `deploy` folder at the root level of your repository and put there the `jar` and `war` files you want to deploy to the service.
```
Comment (jonnilundy) [inline ui/src/pages/docs/other/liferay.md]: flows a bit smoother with "At the root level of your repository, create a `deploy` folder and put the `jar` and `war` files you want to deploy to the service"

File: ui/src/pages/docs/other/liferay.md [hunk 2]
```diff
@@ -57,6 +57,20 @@ The `id` for your services are uniquely determined by you.
...
+
+Under the hood, those applications will be copied into the `$LIFERAY_HOME/deploy` folder inside the service, so they will be automatically deployed on portal's startup.
+
+<aside>
+This deploy folder convention is the opinionated way of deploying applications in a managed Liferay, so WeDeploy follows the same structure for the sake of simplicity.
```
Comment (jonnilundy) [inline ui/src/pages/docs/other/liferay.md]: add an extra line break on both sides of text within the `<aside>` element

File: ui/src/pages/docs/other/liferay.md (hunk 1 above)
Comment (mdelapenya) [inline ui/src/pages/docs/other/liferay.md]: LGTM 👍

**Comment by zenorocha** [comment]
Thanks @mdelapenya!

I forgot about this PR and ended up writing this section again, let me know what you think: https://wedeploy.xyz/docs/deploy/deploying-liferay-dxp.html

**Pull request closed without merging**